### PR TITLE
Update release v0.7.0

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Add the following code to the `<dependencies>` section of your project:
 Add the following line to the `dependencies` section of your `build.gradle`:
 
 ```groovy
-implementation 'com.meilisearch.sdk:meilisearch-java:0.6.0'
+implementation 'com.meilisearch.sdk:meilisearch-java:0.7.0'
```
Comment (brunoocasali) [inline README.md:66]: @alallema https://github.com/meilisearch/meilisearch-java/blob/main/.code-samples.meilisearch.yaml#L375 you need to change this file also, right? (I was looking at the contributing file)

**Review by brunoocasali — COMMENTED** [review]
Hey @alallema I let a comment about the code samples!

**Review by brunoocasali — APPROVED** [review]
🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥 🔥

**Comment by alallema** [comment]
bors merge

**Pull request merged**